github/filecoin-project.yml

```
@@ -4434,7 +4435,8 @@ repositories:
         content: >
           # Please see the documentation for all configuration options:
 
-          # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+          #
+          https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

I'm assuming this is accidental. I'm a little surprised this change wasn't reflected in the terraform plan though - I'll look into it.

```suggestion
          # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

---

Ah, that was added by the fix workflow! 

https://github.com/filecoin-project/github-mgmt/pull/115/commits/2b179cccc0dce2da27d01432dec8ae3aa05af565 

I'll figure out what's going on there.

---

Apparently that's correct 🤷 yaml parser interprets both versions the same. I'll have a look at the formatter config to see why it proposes this (probably because of the line length) but for now I just merged the fix into master and now I see it already made its' way back here. Sorry for the noise. I'm going to resolve this conversation.